README.md

```diff
@@ -5,70 +5,63 @@
 An HTTP web server written from scratch in Rust (WIP).
 
 ## Documentation
+
 https://docs.rs/blitzkrieg/latest/blitzkrieg/
 
 ## Installation
-```cargo add blitzkrieg```
+
+`cargo add blitzkrieg`
 
 ## Usage
-### Hello world example
+
+### [Hello world](examples/hello_world.rs)
+
 ```rust
 use blitzkrieg::http::{Request, Response};
 use blitzkrieg::server::Server;
 
 fn main() {
-    let server = Server::new("127.0.0.1:7878", Box::new(handler));
+    let server = Server::new("127.0.0.1:7818", Box::new(handler));
```

Comment:
Any particular reason for choosing 7818 over 7878? :D

Reply:
No, My port was busy when running it, So I needed to change it 😁